style.css

```diff
@@ -0,0 +1,817 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;600&display=swap");
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  padding-bottom: 2em;
+
+  --shadow: 0 2px 10px -1px rgb(0 0 0 / 80%);
+  --shadow-accent: 0 2px 50px -1px rgb(0 0 0 / 80%);
+  --color-primary: #36b37f;
+}
```

Comment:
:point_right: *This change is applicable to all the sections in your mobile version*

- Hey, your design is having a horizontal scroll on the mobile version and this has made your design inconsistent compared to the Figma design. Please, let's try to fix this issue. See the image below:

*your design*
![Screenshot from 2023-01-19 18-37-07](https://user-images.githubusercontent.com/59397159/213519423-051a9f9c-b063-4443-9523-52443262c77b.png)

*Figma design*
![Screenshot from 2023-01-19 18-39-57](https://user-images.githubusercontent.com/59397159/213519599-749715d5-4f7c-4ccb-8ab2-a75f485ec6b3.png)